sql/internal/sqlx/sqlx.go

```diff
@@ -216,6 +216,8 @@ type Builder struct {
 	bytes.Buffer
 	QuoteChar byte    // quoting identifiers
 	Schema    *string // schema qualifier
+	Indent    string  // indentation prefix
```

Comment:
Usually a prefix is something else than the indention. E.g. I can have a prefix of '+' and indent '  ': 

```
+  create table t (
+    t int
+  )
```

Not sure if we want that as well to mimic `json.MarshalIndent` behavior.

Reply:
The comment is wrong. I'll change it